python/employee-scheduling/src/employee_scheduling/routes.py

```diff
@@ -0,0 +1,61 @@
+from timefold.solver import SolverManager, SolverFactory, SolutionManager
+from timefold.solver.config import (SolverConfig, ScoreDirectorFactoryConfig,
+                                    TerminationConfig, Duration)
+from fastapi import FastAPI
+from fastapi.staticfiles import StaticFiles
+
+from .domain import EmployeeSchedule, Shift
+from .constraints import scheduling_constraints
+from .demo_data import DemoData, generate_demo_data
+
+
+solver_config = SolverConfig(
+    solution_class=EmployeeSchedule,
+    entity_class_list=[Shift],
+    score_director_factory_config=ScoreDirectorFactoryConfig(
+        constraint_provider_function=scheduling_constraints
+    ),
+    termination_config=TerminationConfig(
+        spent_limit=Duration(seconds=30)
+    )
+)
+
+solver_manager = SolverManager.create(SolverFactory.create(solver_config))
+solution_manager = SolutionManager.create(solver_manager)
+
+app = FastAPI(docs_url='/q/swagger-ui')
+data_sets: dict[str, EmployeeSchedule] = {}
+
+
+@app.get("/demo-data")
+async def demo_data_list() -> list[DemoData]:
+    return [e for e in DemoData]
+
+
+@app.get("/demo-data/{dataset_id}",  response_model_exclude_none=True)
```

Comment:
It looks like we always return the `SMALL` dataset

Reply:
Well, there is no LARGE dataset defined for the employee scheduling Java quickstart; note `generateDemoData` takes no arguments: https://github.com/TimefoldAI/timefold-quickstarts/blob/0edf2339ca75726f068153b4b496385f6b7d7be1/use-cases/employee-scheduling/src/main/java/org/acme/employeescheduling/rest/DemoDataGenerator.java#L55 ; the `DemoData` is only there for ceremony (namely, making it work with the shared logic which expects an enum); `SMALL == LARGE`.